src/Microsoft.AspNet.Mvc/RazorPreCompileModule.cs

```diff
@@ -84,6 +76,27 @@ public void AfterCompile(IAfterCompileContext context)
         {
         }
 
+        // REVIEW: Welcome back to life delegating SP!!!
```

Comment:
put another way: we were using `HostingServices.Create()` and it was less evil / ugly than this